packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -474,26 +496,28 @@ contract DeployImplementations is Script {
     // Deploy and initialize a proxied OPStackManager.
     function createOPSMContract(
         DeployImplementationsInput _dii,
-        DeployImplementationsOutput,
-        OPStackManager.Blueprints memory blueprints,
-        string memory release,
-        OPStackManager.ImplementationSetter[] memory setters
+        DeployImplementationsOutput _dio,
+        OPStackManager.Blueprints memory _blueprints,
+        string memory _release,
+        OPStackManager.ImplementationSetter[] memory _setters
     )
         internal
         virtual
         returns (OPStackManager opsmProxy_)
     {
-        SuperchainConfig superchainConfigProxy = _dii.superchainConfigProxy();
-        ProtocolVersions protocolVersionsProxy = _dii.protocolVersionsProxy();
         ProxyAdmin proxyAdmin = _dii.superchainProxyAdmin();
 
         vm.startBroadcast(msg.sender);
         Proxy proxy = new Proxy(address(msg.sender));
-        OPStackManager opsm = new OPStackManager(superchainConfigProxy, protocolVersionsProxy);
+
+        deployOPContractsManagerImpl(_dii, _dio);
+        OPStackManager opsmImpl = _dio.opsmImpl();
```

Comment:
same comment around `broadcast` vs. `startBroadcast`

Reply:
Added in: https://github.com/ethereum-optimism/optimism/pull/12035/commits/c131a31a0cd4683f239d8a59df7fc6bd70f271ed